.github/workflows/tests.yml

```diff
@@ -44,10 +44,17 @@ jobs:
           git config --global user.name "Tar Antool"
 
       - name: Install Tarantool
+        if: matrix.tarantool-version != '2.10'
         uses: tarantool/setup-tarantool@v1
         with:
           tarantool-version: '${{ matrix.tarantool-version }}'
 
+      - name: Install Tarantool 2.10
+        if: matrix.tarantool-version == '2.10'
```

Comment:
If there will be Tarantool 2.11, it will be in pre-release installer, so I think `2.10` is not the best label for it.